Libraries/LibWeb/CSS/Parser/PropertyParsing.cpp

```diff
@@ -1552,6 +1552,15 @@ RefPtr<StyleValue const> Parser::parse_border_value(PropertyID property_id, Toke
         VERIFY_NOT_REACHED();
     }
 
+    auto const make_single_value_shorthand = [&](PropertyID property_id, Vector<PropertyID> const& longhands, ValueComparingNonnullRefPtr<StyleValue const> const& value) {
+        Vector<ValueComparingNonnullRefPtr<StyleValue const>> longhand_values;
+
+        for (size_t i = 0; i < longhands.size(); i++)
+            longhand_values.append(value);
```

Comment:
I think you should be able to do `longhand_values.resize_with_default_value(longhands.size(), value);` here